### PR TITLE
[REG-1985] Reorganize docs site

### PR DESCRIPTION
**Pull request by abeizer**
https://www.loom.com/share/2dc0a5506c4f48fca5e60fe5256988b7

---

Find the pull request instructions [here](https://www.notion.so/regressiongg/Pull-Request-Process-0d57a7eb582a446983edfacafa406f1e?pvs=4)

Every reviewer and the owner of the PR should consider these points in their request (feel free to copy this checklist so you can fill it out yourself in the overall PR comment)

- [ ] The code is extensible and backward compatible
- [ ] New public interfaces are extensible and open to backward compatibility in the future
- [ ] If preparing to remove a field in the future (i.e. this PR removes an argument), the argument stays but is no longer functional, and attaches a deprecation warning. A linear task is also created to track this deletion task.
- [ ] Non-critical or potentially modifiable arguments are optional
- [ ] Breaking changes and the approach to handling them have been verified with the team (in the Linear task, design doc, or PR itself)
- [ ] The code is easy to read
- [ ] Unit tests are added for expected and edge cases
- [ ] Integration tests are added for expected and edge cases
- [ ] Functions and classes are documented
- [ ] Migrations for both up and down operations are completed
- [ ] A documentation PR is created and being reviewed for anything in this PR that requires knowledge to use
- [ ] Implications on other dependent code (i.e. sample games and sample bots) is considered, mentioned, and properly handled
- [ ] Style changes and other non-blocking changes are marked as non-blocking from reviewers

### CODE REVIEW

**Comment by abeizer** [comment]
I'm aware there are broken links. I wish building the site would warn you like CircleCI does... anyway, fixing these while I get reviews.

**Review by batu — APPROVED** [review]
This looks good! 

My only concern is that "Installing Regression Games" is such a core page, that I don't think we should publish our docs with it being under construction. Given that we have new customers onboarding, it might just look bad. 

Can we either hide that page, or copy some of the content from: https://docs.regression.gg/tutorials/building-your-first-bot#import-the-regression-games-unity-bots-package?

File: docs/core-concepts/bot-sequences/introduction-to-bot-sequences-and-segments.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+sidebar_label: 'Introduction to Bot Sequences and Segments'
+---
+
+# Introduction to Bot Sequences and Segments
+
+import Placeholder from '../../partials/_placeholder.mdx';
+
+<Placeholder />
```
Comment (addisonbgross) [inline docs/core-concepts/bot-sequences/introduction-to-bot-sequences-and-segments.mdx:9]: I think it could be nice to have the label: `Getting Started with Bot Sequences and Segments`
This way the label would match up with the first page of each section
image

**Comment by abeizer** [comment]
> This looks good!
> 
> My only concern is that "Installing Regression Games" is such a core page, that I don't think we should publish our docs with it being under construction. Given that we have new customers onboarding, it might just look bad.
> 
> Can we either hide that page, or copy some of the content from: https://docs.regression.gg/tutorials/building-your-first-bot#import-the-regression-games-unity-bots-package?

I'm working on this right now (https://linear.app/regression-games/issue/REG-1986/create-an-installation-setup-guide-for-rg).
Not sure whether Aaron has a preference, but it sounds like we'll be focusing on our docs site for the rest of the week so these pages shouldn't stay empty for too long.

File: docs/core-concepts/bot-sequences/introduction-to-bot-sequences-and-segments.mdx (hunk 1 above)
Comment (abeizer) [reply docs/core-concepts/bot-sequences/introduction-to-bot-sequences-and-segments.mdx]: Updated

**Review by addisonbgross — APPROVED** [review]
Approved with the assumption that we'll be filling out those stubbed pages

**Review by vontell — CHANGES_REQUESTED** [review]
Just a two things:
* I think we should make a temporary `mvp` branch that we merge all the changes into. As mentioned, we will get a lot of docs ironed out soon, but I don't think we should be deploying these docs yet, and so it would be easy to just work off a separate branch until we are ready.
* This will break a lot of links that we have out there in the wild. After discussing this with Abby, we decided we will do this as a last step potentially for the mvp branch, but just for the important links.

**Review by vontell — APPROVED** [review]
